app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,38 @@
+class PasswordResetsController < ApplicationController
+  layout "home"
+
+  def new
+  end
+
+  def create
+    user = User.find_by_email params[:email]
+
+    if user
+      user.generate_password_reset_data
+      PasswordResetsMailer.send_instructions(user).deliver_now
```

Comment:
Maybe we should set this up on delayed_job ? 

:shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: